applications/nrf_desktop/doc/wheel.rst

```diff
@@ -1,7 +1,48 @@
 .. _wheel:
 
-Wheel module
+wheel module
 ############
 
-Wheel module is responsible for generating events related to the rotation of
+``wheel`` module is responsible for generating events related to the rotation of
 the mouse wheel.
+
+Configuration
+*************
+
+Module is enabled by :option:`CONFIG_DESKTOP_WHEEL_ENABLE`.
+
+For detecting rotation ``wheel`` uses Zephyr QDEC driver. Module can be enabled
+only when QDEC is configured in DTS (:option:`HAS_DTS_QDEC` is available).
+
+QDEC DTS configuration specifies how many steps are done during one full angle
+rotation. Sensor reports rotation data using angle.
+``wheel`` configuration option
+:option:`CONFIG_DESKTOP_WHEEL_SENSOR_VALUE_DIVIDER` is used to convert obtained
+value of angle into value passed to HID data subscriber.
```

Comment:
Mention that subscriber receives data in steps ?

Reply:
I would live it as is. Adding more explanation would require explaining more where these data go and so on, complicating this doc.

Reply:
ok